content_dev/docs/tutorials/atmosphere-pro/_index.md

```diff
@@ -2,7 +2,7 @@
 layout: codelab-list # The layout for a codelab list (think of this as a title page for the code lab)
 
 title: "atmosphere pro" # Title of the codelab
-lead: Learn how the atmosphere pro app works # Description of the codelab
+lead: Make Contributions to our P2P File Transferring App # Description of the codelab
```

Comment:
Revert this change please

Reply:
Hopefully this is it - https://github.com/atsign-foundation/docs.atsign.com/pull/124/commits/5c62ffe18646aca6ee1aff897d834d03ba153ee5